tests/com/github/bazelbuild/rules_jvm_external/javadoc/ResourceTest.java

```diff
@@ -0,0 +1,52 @@
+package com.github.bazelbuild.rules_jvm_external.javadoc;
+
+import com.google.common.collect.ImmutableMap;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+
+import static com.github.bazelbuild.rules_jvm_external.JarUtils.createJar;
+import static com.github.bazelbuild.rules_jvm_external.JarUtils.readJar;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.junit.Assert.assertEquals;
+
+public class ResourceTest {
+  @Rule
+  public TemporaryFolder temp = new TemporaryFolder();
+
+  @Test
+  public void shouldIncludeResourceFiles() throws Exception {
```

Comment:
maybe 2 tests?
1 for no nested directories; another for tested directories?

Reply:
Spoke in person. Just added a note for the untested behavior since we just need top level at the moment.